tasks/github_tasks.py

```diff
@@ -188,3 +191,61 @@ def get_token_from_app(_, app_id_env='GITHUB_APP_ID', pkey_env='GITHUB_KEY_B64')
     from .libs.common.github_api import GithubAPI
 
     GithubAPI.get_token_from_app(app_id_env, pkey_env)
+
+
+def _get_teams(changed_files, owners_file='.github/CODEOWNERS') -> List[str]:
+    codeowners = read_owners(owners_file)
+
+    team_counter = defaultdict(lambda: 0)
```

Comment:
just in case, you have a [Counter](https://docs.python.org/3/library/collections.html#collections.Counter) class in collections as well, with the `most_common` already implemented